model/filter.go

```diff
@@ -8,6 +8,10 @@ import (
 	"go.mongodb.org/mongo-driver/bson/primitive"
 )
 
+const (
+	COMPLETED = "completed"
```

Comment:
It's generally a Go convention not to use ALL_CAPS for any variable name including constants (generally standard CamelCase is used).